tests/e2e/blocks/enform.spec.js

```diff
@@ -191,12 +191,12 @@ test.describe('create, use and submit EN Form', () => {
 });
 
 async function createPageWithENForm(page, admin, editor, style) {
-  await createPostWithFeaturedImage({admin, editor}, {
+  await createPostWithFeaturedImage({page, admin, editor}, {
     title: 'Test page with enform (style)',
     postType: 'page',
   });
 
-  await editor.canvas.getByRole('button', {name: 'Add default block'}).click();
+  await page.locator('button[aria-label="Add block"]').click();
```

Comment:
Why not use the `searchAndInsertBlock` function here? Although, we are about to remove this block so maybe not worth the effort